packages/cli/test/functional/test_site_convert/expected/Home.html

```diff
@@ -73,6 +73,8 @@
     </footer>
   </div>
 </body>
+
```

Comment:
You could use https://mozilla.github.io/nunjucks/templating.html#whitespace-control to remove the blank line diffs + make the html a little tighter.